rancher2/resource_rancher2_cluster_sync.go

```diff
@@ -75,7 +76,7 @@ func resourceRancher2ClusterSyncCreate(d *schema.ResourceData, meta interface{})
 		}
 	}
 
-	d.SetId(clusterID)
+	d.SetId(clusterSyncPrefixID + clusterID)
```

Comment:
@rawmind0 FYI this is a breaking change. I tried using the latest (master) build of this provider, and I had a dependency on the rancher2_cluster_sync.id, and it's broke now. I think we need to revert this before doing a release.

https://www.terraform.io/docs/extend/best-practices/versioning.html#example-major-number-increments

> Increasing the MAJOR number is intended to signify potentially breaking changes.
> Within Terraform provider development, some examples include:
> * Changing resource ID format
>

Reply:
Moved this to an issue https://github.com/rancher/terraform-provider-rancher2/issues/419